src/Converter/Feature2Mesh.js

```diff
@@ -481,12 +495,59 @@ function featureToExtrudedPolygon(feature, options) {
     return new THREE.Mesh(geom, options.polygonMaterial);
 }
 
+/**
+ * Created Instanced object from on mesh
+ *
+ * @param {THREE.MESH} mesh Model 3D to instanciate
+ * @param {*} count number of instances to create (int)
+ * @param {*} ptsIn positions of instanced (array double)
+ * @returns {THREE.InstancedMesh} Instanced mesh
+ */
+function createInstancedMesh(mesh, count, ptsIn) {
+    var instancedMesh = new THREE.InstancedMesh(mesh.geometry, mesh.material, count);
+    var index = 0;
+    for (let i = 0; i < count * 3; i += 3) {
+        const mat = new THREE.Matrix4();
+        mat.setPosition(ptsIn[i], ptsIn[i + 1], ptsIn[i + 2]);
+        instancedMesh.setMatrixAt(index, mat);
+        index++;
+    }
+
+    instancedMesh.instanceMatrix.needsUpdate = true;
+
+    return instancedMesh;
+}
+
+/**
+ * Convert a [Feature]{@link Feature} of type POINT to a Instanced meshes
+ *
+ * @param {Object} feature
+ * @returns {THREE.Mesh} mesh or GROUP of THREE.InstancedMesh
+ */
+function pointsToInstancedMeshes(feature) {
+    const ptsIn = feature.vertices;
+    const count = feature.geometries.length;
+    const modelObject = feature.style.point.model.object;
+
+    if (modelObject instanceof THREE.Mesh) {
+        return createInstancedMesh(modelObject, count, ptsIn);
+    } else if (modelObject instanceof THREE.Object3D) {
+        const group = new THREE.Group();
+        // Get independent meshes from more complexe object
+        var meshes = separateMeshes(modelObject);
+        meshes.forEach(mesh => group.add(createInstancedMesh(mesh, count, ptsIn)));
+        return group;
+    } else {
+        console.error('Format not supported');
```

Comment:
Could you specify a bit more context in the error message so that users know the unsupported format relates the `style.point.model.object` they set ?

Reply:
```suggestion
            console.error(`The format of the model object provided in the feature style is not supported: ${feature.style.point.model.object.type}`);

```

Reply:
Like that
```suggestion
        console.error('The format of the model object provided in the feature style (feature.style.point.model.object) is not supported. Only THREE.Mesh or THREE.Object3D are supported.');

```

Reply:
But it should be good to handle the error. Leaving it like that will make the following `mesh.feature = feature;` crashed.
What could be done in this case in your opinion ?

Reply:
I was thinking of setting a default mesh in this case, but perhaps it would be better to just throw an error (with `throw new Error('...')` in order to stop code execution and still get the error message in the console. WDYT ?

Reply:
If it's not too hard to implement: what about logging a warning (or an error?) and displaying the geometries as simple points without style? WDYT?

Reply:
```suggestion
        throw new Error('The format of the model object provided in the feature style (feature.style.point.model.object) is not supported. Only THREE.Mesh or THREE.Object3D are supported.');

```
Then:

```

 try {
    mesh = pointsToInstancedMeshes(feature);
    mesh.isInstancedMesh = true;
} catch (e) {
    console.error(e);
    mesh = featureToPoint(feature, options);
}
```